portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,7 +14,18 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.io.*;
```

Comment:
The Java style guide [recommends against wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) because they clutter the namespace.

Reply:
deleted